extensions/amp-facebook-comments/1.0/facebook-comments.type.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/** @externs */
+
+ /**
+ * @typedef {{
+ *  data-href: string,
```

Comment:
React props tend to be lowerCamelCase

Reply:
I think `data-` attributes don't need to be in a type definition. They are sort of meant as a "free-form map keys".